clientd3d/dialog.c

```diff
@@ -629,7 +629,7 @@ void SetDescParams(HWND hParent, int flags)
 * DisplayDescription:  Display given object's description.
 *   Caller must ensure that obj is not deleted while dialog is up.
 *   flags gives info on what user can do with dialog.
-*   hDescParent global is used as parent for this dialog.
```

Comment:
It looks like now hDescParent is no longer used, and further, the function SetDescParams no longer works.  SetDescParams is called from a lot of places in the client.  You'll need to test them all to make sure they still work.